manifests/user.pp

```diff
@@ -13,63 +13,94 @@
 #   `1` - CALLBACK
 #
 #   Controls the rights of the user to be created.
+# @param enable
+#   Should this user be enabled?
 # @param user_id
-#   The user id of the user to be created. Should be unique from existing users. On
-#   SuperMicro IPMI, user id 2 is reserved for the ADMIN user.
+#   The user id of the user to be created. Should be unique from existing users.
+#   On SuperMicro IPMI, user id 2 is reserved for the 'ADMIN' username.
+#   On ASUS IPMI, user id 2 is reserved for the 'admin' username.
 #
 define ipmi::user (
-  Variant[Sensitive[String[1]], String[1]] $password,
   String $user     = 'root',
   Integer $priv    = 4,
+  Boolean $enable  = true,
   Integer $user_id = 3,
+  Optional[Variant[Sensitive[String[1]], String[1]]] $password = undef,
```

Comment:
If we are going to change the order of the parameters, could we change the order of the puppet doc comments to be the same?

Reply:
Done.  I re-arranged these to make puppet-lint happy.